tests/integration/test_celeryadapter.py

```diff
@@ -250,7 +250,7 @@ def test_get_running_queues(self):
         This should return an empty list.
         """
         result = celeryadapter.get_running_queues("merlin_test_app", test_mode=True)
-        assert result == []
+        assert not result
```

Comment:
I actually changed this one as our linter was complaining about it. It specifically wants us to use the `not result` syntax rather than checking for an empty list/set